### ISSUE TITLE
Consider dropping bfdbg

### ISSUE THREAD

**Issue by condret** [radareorg/radare2#18314]
Now that we have esil, a cleaner way to emulate and debug bf would be esil. so maybe we no longer need bfdbg

**Comment by trufae** [comment]
Indeed. But its a simple and friendly plugin to be taken as example when implementing a debug plugin. What about moving to extras? Anyway bfdbg is setting up devices and such too (screen, leyboard..) is there any test for it?

**Comment by condret** [comment]
moving it to extras sounds good

**Comment by trufae** [comment]
That helped us find some really scary bugs in the arch refactoring so imho its a good plugin to keep in order to test debugging and arxh swapping
